eth2deposit/key_handling/key_derivation/word_lists/english.txt

```diff
@@ -0,0 +1,2048 @@
+abandon
```

Comment:
why copied? usually any library will have functions of key derivations etc.

Reply:
I have copied and modified the code from eth-deposit-cli. It was like that in the repo.